ui/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -185,12 +187,13 @@ export default class SettingsTab extends PureComponent {
                 </label>
               </div>
               <div className="settings-tab__radio-button">
-                <input
-                  type="radio"
+                <RadioButton
+                  name="preferred-currency"
```

Comment:
should this have a different `name` than the button for the native primary currency?